frontend/packages/ux-editor/src/components/config/editModal/EditDataModelBindings.tsx

```diff
@@ -22,7 +22,9 @@ export const EditDataModelBindings = ({
   handleComponentChange,
   renderOptions,
 }: EditDataModelBindingsProps) => {
-  const dataModel = useSelector((state: IAppState) => state.appData.dataModel.model);
+  const { org, app } = useParams();
+  const datamodelQuery = useDatamodelQuery(org, app);
```

Comment:
Isn't it easier to use object deconstruction?

Reply:
In general I have chosen not to do it because variable names like `data`, `isLoading` and `isSuccess` etc. doesn't say anything about what kind of data they are associated with. But I agree that it would be better in this particular case, fixing it right away.